editor/src/messages/tool/tool_messages/path_tool.rs

```diff
@@ -397,10 +397,8 @@ impl PathToolData {
 			}
 			self.drag_start_pos = input.mouse.position;
 			self.previous_mouse_position = document.metadata().document_to_viewport.inverse().transform_point2(input.mouse.position);
-			shape_editor.select_connected_anchors(document, layer, input.mouse.position);
 
 			responses.add(DocumentMessage::StartTransaction);
-
```

Comment:
Please carefully review the diff of your changes before staging and committing them so you're not including random changes to other lines in the file, like the addition and removal of blank lines, unless those are intentional (but I'm commenting this because I'm assuming these aren't intentional and are thus spammy and need to take up all our time to restore them during code review).